server/server.js

```diff
@@ -7,8 +7,12 @@ import http from 'http';
 import { fileURLToPath } from 'url';
 import require_auth from './middleware/Auth.js';
 import require_auth_proxylinks from './middleware/proxyLinksAuth.js';
-import { updateSocket } from './SocketManager/socketmanager.js';
+import {
+  socket_user_map,
+  updateSocket,
+} from './SocketManager/socketmanager.js';
 import { Server } from 'socket.io';
+import { logger, log } from './logger.js';
 const server = http.createServer(app);
```

Comment:
The duplicate `app.get('/*', ...)` route handler should be removed to avoid redundancy. This will streamline the code and prevent potential conflicts.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*